### PR TITLE
Added flows for un-manage cluster

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/commons#797

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import datetime
+import json
+import os
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils.time_utils import now as tendrl_now
+from tendrl.monitoring_integration.flows import alert_utils
```
Comment (r0h4n) [inline tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py]: use https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/alert_utils.py

File: tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        if not os.path.exists(archive_base_path):
+            os.makedirs(str(archive_base_path))
+        archive_path = "%s/%s_%s" % (
+            archive_base_path,
+            integration_id,
+            str(datetime.datetime.now().isoformat())
+        )
+        resource_path = "%s/clusters/%s" % \
+                        (carbon_utils.get_data_dir_path(), integration_id)
+        ret_val = os.system("mv %s %s" % (resource_path, archive_path))
```
[the remark this answers is not in the document]
Comment (shtripat) [reply tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py]: ok

File: tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                }
+            )
+            return False
+
+        # Raise a notification mentioning the archive data location
+        self._emit_event(integration_id, archive_path)
+
+        return True
+
+    def _emit_event(self, integration_id, archive_path):
```
[the remark this answers is not in the document]
Comment (shtripat) [reply tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py]: The commons one contains sds_name setting which we dont need in our case so have this local one.

File: tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py (hunk 1 above)
Comment (shtripat) [reply tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py]: ok . will try that

File: tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (shtripat) [reply tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py]: Done

File: tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py (hunk 1 above)
Comment (shtripat) [reply tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py]: After re-factoring this is clean now

**Comment by shtripat** [comment]
@r0h4n re-based on top of re-factored monitoring-integration code now

**Pull request merged**